products/graylog.md

```diff
@@ -32,6 +32,13 @@ auto:
 # eol(x) = releaseDate(x+2)
 # Note: for release < 4.0, used releaseDate(x+1) as it looked more accurate
 releases:
+  - releaseCycle: "7.0"
+    releaseDate: 2025-11-03
+    eol: false
```

Comment:
```suggestion
    eol: 2026-11-03
```

According to https://go2docs.graylog.org/current/setting_up_graylog/supported_versions.htm.